README.md

```diff
@@ -1,2 +1,192 @@
+REDIS_HOST := "my.redis-server.com"
+REDIS_PORT := 6379
+// Create a Redis client
+redisClient := redis.NewClient(&redis.Options{Addr: REDIS_HOST + ":" + REDIS_PORT,})
+```
+</td>
+</tr>
+<tr>
+<td width="50%">
+
+```go
+package redis
+
+import (
+	"context"
+	"github.com/momentohq/client-sdk-go/auth"
+	"github.com/momentohq/client-sdk-go/config"
+	"github.com/momentohq/client-sdk-go/momento"
+	"github.com/momentohq/momento-go-redis-client/momento-redis"
+)
+
+credential, _ := auth.NewEnvMomentoTokenProvider("MOMENTO_AUTH_TOKEN")
+cacheClient, _ := momento.NewCacheClient(config.LaptopLatest(), credential, 60*time.Second)
+// create cache; it resumes execution normally incase the cache already exists and isn't exceptional
+cacheClient.CreateCache(context.Background(), &momento.CreateCacheRequest {CacheName : "default_cache"})
+redisClient := momento_redis.NewMomentoRedisClient(cacheClient, "default_cache")
+```
+
+</td>
+</tr>
+</table>
+
+**NOTE**: The Momento `momento-redis` implementation currently supports simple key/value pairs (`GET`, `SET`, `SETNX`, `DEL`, `EXPIRE`, `TTL`),
+and doesn't support statefulCmdable APIs. We will continue to add support for additional Redis APIs in the future; 
+for more information see the [Current Redis API Support](#current-redis-api-support) section later in this doc.
+
+## Current Redis API Support
+
+This library supports the most popular Redis APIs, but does not yet support all Redis APIs. We currently support the most
+common APIs related to string values (GET, SET, etc.). We will be adding support for additional
+APIs in the future. If there is a particular API that you need support for, please drop by our [Discord](https://discord.com/invite/3HkAKjUZGq)
+or e-mail us at [support@momentohq.com](mailto:support@momentohq.com) and let us know!
+
+In the meantime, if you call a method from the `momento-redis` API that we do not yet support, you will get a panic for 
+`UnsupportedOperationError`; letting you know that the method is not implemented yet.
+
+## Installation
+
+```bash
+go get github.com/momentohq/momento-go-redis-client@v0.1.8
+```
+
+## Examples
+
+### Prerequisites
+
+To run these examples, you will need a Momento auth token. A Momento auth token is required, you can generate one using the [Momento Console](https://console.gomomento.com)
+
+The examples will utilize your auth token via the environment variable `MOMENTO_AUTH_TOKEN` you set.
+
+### Basic Example
+
+In the [`examples/basic`](./examples/basic) directory, you will find a simple CLI app that does some basic sets and gets
+on string values. You can also run the tests against your Redis server by providing ```-useRedis``` flag along 
+with ```-host x.x.x.x``` and ```-port xxxx```.
+
+You can run the example via `go run`.
+
+Here's an example run against Momento Cache:
+
+```bash
+cd examples/basic
+export MOMENTO_AUTH_TOKEN=<your momento auth token goes here>
+go run main.go -cacheName cache -authToken $MOMENTO_AUTH_TOKEN
```

Comment:
I don't think it's a good idea from a security perspective to have users pass an auth token on the command line. Most of our other examples read the environment variable directly, and you should probably do that in this example as well.